repose-aggregator/artifacts/extensions-filter-bundle/src/config/docs/DEPENDENCIES.txt

```diff
@@ -17,17 +17,17 @@ outlined in CONTRIBUTORS.txt.
-com.fasterxml.jackson.core                          jackson-annotations       2.8.9            Apache License Version 2.0  http://wiki.fasterxml.com/JacksonDownload
-com.fasterxml.jackson.core                          jackson-core              2.8.9            Apache License Version 2.0  http://wiki.fasterxml.com/JacksonDownload
-com.fasterxml.jackson.core                          jackson-databind          2.8.9            Apache License Version 2.0  http://wiki.fasterxml.com/JacksonDownload
-com.google.guava                                    guava                     18.0             Apache License Version 2.0  https://github.com/google/guava
+com.fasterxml.jackson.core                          jackson-annotations       2.9.8            Apache License Version 2.0  http://wiki.fasterxml.com/JacksonDownload
+com.fasterxml.jackson.core                          jackson-core              2.9.8            Apache License Version 2.0  http://wiki.fasterxml.com/JacksonDownload
+com.fasterxml.jackson.core                          jackson-databind          2.9.8            Apache License Version 2.0  http://wiki.fasterxml.com/JacksonDownload
+com.google.guava                                    guava                     27.0.1-jre       Apache License Version 2.0  https://github.com/google/guava
 commons-codec                                       commons-codec             1.10             Apache License Version 2.0  http://commons.apache.org/proper/commons-codec/
 commons-io                                          commons-io                2.4              Apache License Version 2.0  http://commons.apache.org/proper/commons-io/
 commons-pool                                        commons-pool              1.6              Apache License Version 2.0  http://commons.apache.org/proper/commons-pool/
 com.rackspace.identity.components.attribute-mapper  mapper-core               2.2.1            Apache License Version 2.0  https://github.com/rackerlabs/attributeMapping
-com.rackspace.papi.components.api-checker           checker-core              2.6.1            Apache License Version 2.0  https://github.com/rackerlabs/api-checker/blob/api-checker-2.5.1/LICENSE.txt
-com.rackspace.papi.components.api-checker           checker-util              2.6.1            Apache License Version 2.0  https://github.com/rackerlabs/api-checker/blob/api-checker-2.5.1/LICENSE.txt
-com.typesafe.akka                                   akka-actor_2.11           2.3.12           Apache License Version 2.0  https://github.com/akka/akka
+com.rackspace.papi.components.api-checker           checker-core              2.7.0            Apache License Version 2.0  https://github.com/rackerlabs/api-checker/blob/api-checker-2.5.1/LICENSE.txt
+com.rackspace.papi.components.api-checker           checker-util              2.7.0            Apache License Version 2.0  https://github.com/rackerlabs/api-checker/blob/api-checker-2.5.1/LICENSE.txt
```

Comment:
The license link should probably be bumped to the same version we reference.

Reply:
Good catch. Done.